ansible/roles/setup_course/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Copy setup-course dockerfile
```

Comment:
Our other tasks have all lower case characters for names. The linter currently doesn't check for this convention so we probably need to update it in a separate PR.